From: Takashi Iwai <tiwai@suse.de>
Subject: Re-export snd_cards for kABI compatibility
Patch-mainline: Never, kABI compatibility
References: bsc#1051510

Re-export snd_cards object only for kABI compatibility with SLE15.
It's not declared in the header.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/core/init.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/sound/core/init.c
+++ b/sound/core/init.c
@@ -49,7 +49,8 @@ static const struct file_operations snd_
 
 /* locked for registering/using */
 static DECLARE_BITMAP(snd_cards_lock, SNDRV_CARDS);
-static struct snd_card *snd_cards[SNDRV_CARDS];
+struct snd_card *snd_cards[SNDRV_CARDS];
+EXPORT_SYMBOL(snd_cards); /* XXX exported only for SLE15 kABI compatibility */
 
 static DEFINE_MUTEX(snd_card_mutex);
 
